src/fn/fn-slider.scss

```diff
@@ -59,10 +68,10 @@ $slider-height: 0.5rem;
 
   &__track-range {
     @include fn-reset();
-    @include fn-set-position-left(0);
 
     z-index: 2;
-    width: 100%;
+    left: -$slider-wrapper-offset;
+    right: -$slider-wrapper-offset;
```

Comment:
for consistency, please use the mixins `fn-set-position-right` and `fn-set-position-left`. Thank you!

Reply:
Discussed with Denis, ignore my comment

Reply:
No need since values are equal